examples/python/setup.py

```diff
@@ -0,0 +1,32 @@
+import codecs
+import os
+
+import setuptools
+
+
+def read(filename):
+    """Read and return `filename` in root dir of project and return string."""
+    here = os.path.abspath(os.path.dirname(__file__))
+    with codecs.open(os.path.join(here, filename), 'r') as openfile:
+        return openfile.read()
+
+
+setuptools.setup(
+    name='liquid_donations',
+    version='0.0.1',
+    description='Liquid Donations Proof of Concept.',
+    long_description=read('README.md'),
+    packages=setuptools.find_packages(),
+    package_data={
+        'static': ['README.md'],
+    },
+    install_requires=[
+        'click',
+        'numpy',
+        'pandas',
```

Comment:
What are these packages?

Reply:
* `click`: Library that lets you make a commandline tool
Makes stuff like this easy:
```bash
$ liquid_donations --help
Usage: liquid_donations [OPTIONS] FILENAME

  Tool for calculating liquid donations.

Options:
  --help  Show this message and exit.
```
* `numpy`: Numerical computing that's super efficient (forks out to C)
* `pandas`: Built on top of numpy to make stuff even easier to work with